### DISCUSSION TITLE
Feature Request: Ability to Skin Non-Button Unit Frames

### DISCUSSION THREAD

**Discussion by ars4l4n** [SFX-WoW/Masque#225] [Issues] [answered]
### Game Flavor

Retail

### Game Version

9.1.5

### Add-On Version

9.1.5

### Describe the Improvement

Correct me if I'm wrong, but as far as I see you can't skin non-button unitframes like the player frame with masque. Although I could imagine that at least for eg the player frame in elvui which is rectangular it shouldn't be much harder/different to achieve that, so I'd like to see an implementation of adding the option to skin some non-button-unitframes.

Thank you for your time

**Comment by StormFX** [comment, marked as answer]
Masque is exclusively a button-skinning engine. There are add-ons that allow for the customization of other frames like unit frames.
